HotelReservation/ConsoleUserInterface/SystemMain.cs

```diff
@@ -28,10 +53,11 @@ public static int Menu(HotelSystem hotelSystem, StepsExecutor stepsExecutor)
                     break;
                 case "1":
                     Console.Clear();
-                    new HotelsDisplay().DisplayHotels(hotelSystem);
-                    var hotelId = new HotelSelect().SelectHotel(hotelSystem);
+                    var hotels = hotelSystem.GetHotels();
+                    DisplayHotels(hotels);
+                    var hotelId = SelectHotel();
                     Console.Clear();
-                    if (hotelId >= 1000)
+                    if (hotelId >= 1000 & hotelId < hotelSystem.LastHotelId)
```

Comment:
Wiedza, że ID zaczyna się od 1000 to nie jest do końca coś, czego powinna być świadoma warstwa interfejsu. GetHotelReservationSteps(hotelId) może zwrócić błąd lub wyjątek jeśli nie ma hotelu z takim ID, jakiego szukasz

Reply:
Fixed